qemu/tests/hugepage_reset.py

```diff
@@ -0,0 +1,74 @@
+from avocado.utils import process
+
+from virttest import env_process
+from virttest import test_setup
+from virttest import utils_test
+from virttest import utils_misc
+from virttest.staging import utils_memory
+
+
+def run(test, params, env):
+    """
+    Re-assign nr-hugepages
+
+    1. Set up hugepage with 1G page size and hugepages=N
+    2. Boot up guest using /mnt/hugetlbfs/ as backend in QEMU CML
+    3. Change the nr_hugepages after the guest boot up (M < N and M > N)
```

Comment:
Let's set N=8, guest memory=4G, M=6 or 10.  Just need to check if  hugepage is set as expected without error.